include/antler/project/dependency.hpp

```diff
@@ -15,10 +15,12 @@
 #include "../system/utils.hpp"
 
 
-namespace antler::project {
+namespace antler::project
+{
```

Comment:
It appears that one distinguishing feature of Microsoft style is placing braces on a new line.

Reply:
Not my fave. Reduces the number of lines/context visible in a given window. I give this a :-1: